examples/room_rt60.py

```diff
@@ -0,0 +1,101 @@
+'''
+Room RT60
+---------
+
+In this example, we create a room with a pre-set reverberation time
+(according to Sabine's formula), and then check how the simulated RIR
+verifies the prediction.
+
+### Example
+
+Simulate a ``10 x 7.5 x 3.2`` room with ``RT60 = 0.5 s``
+
+    python examples/room_rt60.py 10 7.5 3.2 0.5
+
+'''
+import argparse
+import numpy as np
+import pyroomacoustics as pra
+import matplotlib.pyplot as plt
+
+def rt60_analysis(room, mic, src, plot=True, rt60_tgt=None):
+    '''
+    Plot and analyze the RT60 of one of the RIR in the room.
+    '''
+
+    # Remove coefficients that might cause problems with the log
+    h = room.rir[mic][src]
+    T = np.arange(len(h)) / room.fs
+    I = h > 0
+
+    # The power of the impulse response in dB
+    power = 20 * np.log10(np.abs(h[I]))
+    T = T[I]
+
+    # Adjust to location and amplitude of peak power
+    i_max = np.argmax(power)
+    power -= power[i_max]
+    power = power[i_max:]
+    T = T[i_max:]
+
+    # linear fit
+    a, b = np.polyfit(T, power, 1)
+    est_rt60 = - 60 / a
+
+    if plot:
+        # show result
+        plt.plot(T, power)
+        plt.plot(T, a * T + b, '--')
+        plt.plot(T, np.ones_like(T) * -60, '--')
```

Comment:
I'm wondering if here's better to add the offset to the `-60` line? Now it's like this:

![original](https://user-images.githubusercontent.com/4757445/43777432-18578d4e-9a53-11e8-9492-6718bf0e7e2a.png)

But I'm thinking something like this is more representative of a `-60 dB` **drop** rather than `-60 dB`.

![offset](https://user-images.githubusercontent.com/4757445/43777492-41065b76-9a53-11e8-82e3-4fa427c8eec2.png)

Reply:
with the offset that you computed for the orange line

Reply:
Hum, this is an annoying issue. It stems from not having a proper method for computing the empirical T60. I think either is fine now, so I'll go with the more correct version you propose. The correct method will come in a subsequent pull request.